### ISSUE TITLE
Performance on Scannet Val Set

### ISSUE THREAD

**Issue by chronbird** [HuguesTHOMAS/KPConv-PyTorch#52]
Hi @HuguesTHOMAS,

Thank you for the great work. I notice that the Scannet performance is reported on the test set, and I want to ask do you have any results on val set, so that I can compare it to my results to see if I reproduce the experiment successfully? Thanks.

**Comment by HuguesTHOMAS** [comment]
Fig.5 in the paper gives the performances on the validation set. It is not very detailed but I think that should help you see if you reproduce the experiment successfully

**Comment by chronbird** [comment]
Thanks!

**Comment by mutianxu** [comment]
Hi @chronbird ,

Did you successfully reproduce the Scannet Preprocessing of KPConv in PyTorch?

Could you plz share your preprocessing and training script of KPConv under Scannet?

Highly appreciated!

**Comment by HuguesTHOMAS** [comment]
I am sorry but I did not experiment with Scannet in PyTorch. As a matter of fact I did not touch this dataset since the KPConv Paper was published.

Your best option is to try to look at the preprocessing that I used with tensorflow and reproduce it for PyTorch. Looking at what changed for the S3DIS dataset between the two repo should help you.

Sorry I could not be more helpful, and good luck with your experiments.
